src/java.base/unix/classes/sun/net/www/protocol/jar/JarFileFactory.java

```diff
@@ -42,6 +42,9 @@
 /* A factory for cached JAR file. This class is used to both retrieve
  * and cache Jar files.
  *
+ * @crac All JarFile instances that are not referenced from elsewhere are
+ * removed from the cache before a checkpoint.
```

Comment:
Looks OK for a non-public class. But in general, the new tag is used for implementation notes like this and also for the description of the behavior of the internal Resource implementation with more impact on the user. It would be nice to somehow distinguish these two (or more) uses of the tag.